README.md

```diff
@@ -34,13 +34,10 @@ You can change the names and types of the 'attr_' columns in the schema to fit y
 
 ## Instantiating the script
 
-The script takes advantage of Tinybird Analytics' javascript library, so it must be included along side this javascript.
-
-You will also need to create an append token with permissions just with write permissions for the tracker Datasource, that you must include on instantiation, along side the account name you wish to use, the private variable name to use for the tracker and the Tinybird API HOST (which may vary for your account):
+You will need to create an append token with permissions just with write permissions for the tracker Datasource, that you must include on instantiation, along side the account name you wish to use, the private variable name to use for the tracker and the Tinybird API HOST (which may vary for your account):
 
 ```
-<script src="https://cdn.tinybird.co/v1/tinybird.js"> </script>
-<script src="t.js"> </script>
+<script src="https://cdn.tinybird.co/static/js/t.js"></script>
 <script>
   tracker('{{tracker_append_token}}', 'main', '_tracker', '{{api_host}}')
```

Comment:
I'd remove 'api_host' here, it just for us to be able to use different hosts but people should not use this